frontend/src/components/GTerminalShortcut.vue

```diff
@@ -46,18 +46,22 @@ SPDX-License-Identifier: Apache-2.0
     >
       <g-action-button
         icon="mdi-console-line"
-        :disabled="disabled"
+        :disabled="(isAdmin && !canScheduleOnSeed) ||
+          (isShootTarget && isShootHibernated)"
         @click.stop="addTerminalShortcut(shortcut)"
       >
         <template #tooltip>
-          <span v-if="!disabled">
+          <div v-if="isAdmin && !canScheduleOnSeed">
+            Terminal shortcut not available on non-managed seed
```

Comment:
use same/similar text as on the create cluster dialog `Terminals can only be scheduled if the seed is a managed seed`